container/config/noter-backend/launch

```diff
@@ -6,7 +6,8 @@ export NOTER_DB_NAME="noterdb"
 export NOTER_DB_USER=${NOTER_BACKEND_DB_USER}
 export NOTER_DB_PASSWORD=${NOTER_BACKEND_POSTGRES_PASSWORD}
 export NOTER_DB_HOST=${NOTER_BACKEND_DB_HOST}
-export GOOGLE_APPLICATION_CREDENTIALS="/container/secrets/noter-backend-bucket-credential.json"
+export NOTER_GS_MEDIA_BUCKET_NAME=${NOTER_BUCKET}
+export GOOGLE_APPLICATION_CREDENTIALS="/container/secrets/noter-service-account.json"
```

Comment:
Could there be a better (more specific) name for this?

Reply:
Turns out this is one special env variable the Google cloud storage module used here is using to find the needed credentials. Changing the name will break the current setup. So let's keep it now.